### PR TITLE
Update releasing.md

### PR DESCRIPTION
**Pull request by alexec**

### CODE REVIEW

File: docs/developer-guide/releasing.md [hunk 1]
```diff
@@ -28,7 +32,7 @@ Tag UI:
 
 ```bash
 git tag $VERSION
-git push upstream $BRANCH --tags
+git push origin $BRANCH --tags
```
Comment (alexmt) [inline docs/developer-guide/releasing.md]: I think in most cases the remote name is `upstream`, not `origin`. Can you revert this change?

File: docs/developer-guide/releasing.md [hunk 2]
```diff
@@ -57,14 +61,20 @@ make release IMAGE_NAMESPACE=argoproj IMAGE_TAG=$VERSION DOCKER_PUSH=true
...
 * Changelog
+* Binaries (e.g. dist/argocd-darwin-amd64).
 
-## Stable Release
 
-Update Brew formula:
+If GA, update `stable` tag:
+
+```bash
+git tag stable --force && git push origin stable --force
```
Comment (alexmt) [inline docs/developer-guide/releasing.md]: Same here: `git push upsteam stable --force`

**Review by alexmt — CHANGES_REQUESTED** [review]

File: docs/developer-guide/releasing.md (hunk 1 above)
Comment (alexec) [reply docs/developer-guide/releasing.md]: I use origin myself. I'm not precious. I'll create a var for this.

**Review by alexmt — APPROVED** [review]
